argocon-demos/2023-migrating-cicd-from-jenkins-to-argo/jenkins-vs-argo-workflows-comparison/argo-workflow.yaml

```diff
@@ -30,8 +30,8 @@ spec:
       resources:
         requests:
           storage: 1Gi
- # All the steps in this DAG are referencing external templates.
- # This allows us to re-use those templates in other workflows, and also makes this CI workflow quite tidy.
+ # All the steps in this DAG are referencing external WorkflowTemplates.
```

Comment:
This should be changed in the ci-example too (this work is a direct copy of that)